_data/3.10-manual.yml

```diff
@@ -393,8 +393,6 @@
       href: graphs-edges.html
     - text: Pregel
       href: graphs-pregel.html
-    - text: Custom Pregel
-      href: graph-analytics-custom-pregel.html
```

Comment:
Note that the page is still generated:
https://deploy-preview-1038--zealous-morse-14392b.netlify.app/docs/3.10/graph-analytics-custom-pregel.html
I think this is fine, there was at least one prospect who might still want to be able to access the page via a direct link.
If we really want to hide it then we need to add the file to the exclude list in the `_config.yml`.